pkg/apiserver/apiserver.go

```diff
@@ -214,6 +216,7 @@ func installAPIGroup(s *APIServer, c completedConfig) error {
 	cpv1beta2Storage["appliedtogroups"] = appliedToGroupStorage
 	cpv1beta2Storage["networkpolicies"] = networkPolicyStorage
 	cpv1beta2Storage["networkpolicies/status"] = networkPolicyStatusStorage
+	cpv1beta2Storage["networkpolicyevaluation"] = networkPolicyEvaluationStorage
```

Comment:
I might be mistaking, but there is a question that `networkpolicyevaluation` is not related to a pre-provided networkpolicy,  we only know the name after getting the result, so it could not be a subresource? And if we use `networkpolicies/evaluation` as the path, `evaluation` would be considered as the policy name in search.

Reply:
I did some test and found that, since `NetworkPolicyEvaluation` is currently a resource, to use `networkpolicies/evaluation` we have to make evaluation a subresource of `networkpolicies`. But different from status&scale, this `NetworkPolicyEvaluation` is not affiliated to a particular networkpolicy at input, so I could not provide a networkpolicy name for the subresource. I feel like this might have to stay as a separate resource like `appliedtogroups`?